avocado/utils/disk.py

```diff
@@ -121,15 +120,22 @@ def get_disks():
     :returns: a list of paths to the physical disks on the system
     :rtype: list of str
     """
-    json_result = process.run("lsblk --json --paths --inverse")
-    json_data = json.loads(json_result.stdout_text)
-    disks = []
-    for device in json_data["blockdevices"]:
-        disks.append(device["name"])
-        if "children" in device:
-            for child in device["children"]:
-                disks.append(child["name"])
-    return disks
+    try:
+        json_result = process.run("lsblk --json --paths --inverse")
+    except process.CmdError as ce:
+        raise DiskError(f"Error occurred while executing lsblk command: {ce}")
+
+    try:
+        json_data = json.loads(json_result.stdout_text)
+        disks = []
+        for device in json_data["blockdevices"]:
+            disks.append(device["name"])
+            if "children" in device:
+                for child in device["children"]:
+                    disks.append(child["name"])
+        return disks
+    except json.JSONDecodeError as je:
```

Comment:
IMO this `try/except` block should be only around `json_data = json.loads(json_result.stdout_text)` line, because only `json.loads()` will throw `json.JSONDecodeError` error.